libs/db/remotedb/remotedb_test.go

```diff
@@ -14,7 +14,7 @@ import (
 func TestRemoteDB(t *testing.T) {
 	cert := "test.crt"
 	key := "test.key"
-	ln, err := net.Listen("tcp", "0.0.0.0:0")
+	ln, err := net.Listen("tcp", "localhost:0")
```

Comment:
do not bind to all network interfaces (but only to the loopback interface) for tests

For this to work, I had to regenerate server key and cert; see commit (message): 9baec55fd9bb8f32d7d7b50de37e095b49dfb9ec